tests/test_generic.py

```diff
@@ -273,6 +274,17 @@ def test_encode_pdfdocencoding_keyerror():
     assert exc.value.args[0] == "pdfdocencoding"
 
 
+def test_encode_pdfdocencoding_returns_bytes():
```

Comment:
In theory, you should not have to iterate over this list manually - *pytest* provides parametrization support out of the box which we probably want to use here as well: https://docs.pytest.org/en/7.3.x/how-to/parametrize.html